wagn/lib/wagn/application.rb

```diff
@@ -56,9 +56,13 @@ def add_gem_path paths, path, options={}
     def config
       @config ||= begin
         config = super
+
         Cardio.set_config config
+        config.autoload_paths += Dir["#{Cardio.gem_root}/mod/*/lib/**/"]
```

Comment:
Where was this before?  I think it needs to be with the component that triggers module loading on initialization or we have to make engine developers include it (or we repeat ourselves somewhere).